utils/build/docker/agent.Dockerfile

```diff
@@ -30,8 +31,10 @@ otlp_config:\n\
 
 # Proxy conf
 COPY utils/scripts/install_mitm_certificate.sh .
-RUN mkdir -p /usr/local/share/ca-certificates
-RUN ./install_mitm_certificate.sh /usr/local/share/ca-certificates/mitm.crt
-RUN update-ca-certificates
+RUN set -eux;\
+    mkdir -p /usr/local/share/ca-certificates;\
+    ./install_mitm_certificate.sh /usr/local/share/ca-certificates/mitm.crt;\
+    update-ca-certificates;
 
+# Smoke test
```

Comment:
Smoke test ?

Reply:
I guess the execution of the version subcommand is just a check to see if the binary is not completely broken (hence, the clarification about it being a smoke test).